### PR TITLE
Sage integrating celo into the process of fintech web app development

### PR DESCRIPTION
**Pull request by TheKhafre**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    celocli init my-project
+
+This will create a new Celo project in the `my-project` directory, as stipulated in the command.
+
+## Building a Simple Payment App
+
+Now that we have our development environment set up, let's build a simple payment app that allows users to send and receive cUSD. We'll use the Celo SDK's ContractKit and WalletUtils modules to do this.
+
+First, we'll need to create a new Celo wallet for our app:
+
```
Comment (Ishanpathak1) [inline blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md:66]: Add details, like make this file with the name , and files should be in same directory

File: blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    }
+    const kit = ContractKit.newKit('https://forno.celo.org');
+    const account = kit.web3.eth.accounts.create();
+    const contractAddress = await deployContract(kit, account);
+```
+
+This code deploys the reserve contract and logs its address to the console. The `setReserveFraction` method sets the reserve ratio to 1%, which means that for every 100 cUSD in circulation, 1 cUSD is held in reserve. 
+
+Now that we have our contract deployed, let's build a simple frontend that allows users to send and receive cUSD. For this, we'll use the `@celo/dappkit` and the `@celo/contractkit` library.
+
```
Comment (Ishanpathak1) [inline blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md:102]: Add details here also

**Review by Ishanpathak1 — COMMENTED** [review]
Make the required changes

File: blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md (hunk 1 above)
Comment (TheKhafre) [reply blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md:66]: Hi Ishan, what kind of details do you want me to add? is it on the wallet creation?

File: blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md (hunk 2 above)
Comment (TheKhafre) [reply blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md:102]: alright, I'll do just that.

**Review by Ishanpathak1 — COMMENTED** [review]
Mention me on trello card after making the changes

File: blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md (hunk 1 above)
Comment (TheKhafre) [reply blog/sage-integrating-celo-into-the-process-of-fintech-web-app-development/sage-integrating-celo-into-the-process-of-fintech-web-app-development.md:66]: Hi @Ishanpathak1 I have added all the changes you suggested.

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

File: src/data-tutorials/users.tsx [hunk 3]
```diff
@@ -2761,6 +2761,13 @@ const Users: User[] = [
     website: 'blog/tutorials/building-a-celo-voting-system-with-golang',
     tags: ['celosage', 'intermediate', 'smartcontract', 'solidity'], 
   },
+  {
+    title: 'Integrating Celo into the Process of FinTech Web/App Development',
```
Comment (developerkunal) [inline src/data-tutorials/users.tsx:2765]:
```suggestion
    title: 'Integrating Celo into the Process of Web/App Development',
```

**Pull request merged**